src/editors/containers/TextEditor/index.jsx

```diff
@@ -40,6 +42,11 @@ const TextEditor = ({
     learningContextId,
   });
   const editorContent = newContent || initialContent;
+  let staticRootUrl;
+  const ctx = useOptionalLibraryContext();
+  if (isLibrary && ctx !== undefined) {
+    staticRootUrl = `${getConfig().STUDIO_BASE_URL }/library_assets/blocks/${ ctx.sidebarComponentUsageKey }/`;
```

Comment:
Sorry, I should have called this out sooner. We can't assume that the sidebar is open. We need to get the actual usage from the editor state somehow (the redux state most likely), not from the sidebar.

Reply:
Ohh, ok. I will try it

Reply:
Ok, I've updated but CI says: `TypeError: _redux.selectors.app.blockId is not a function`

Reply:
I think your code is right; you just have to update the mock data in the test suite.